docs/quickstart.md

```diff
@@ -3,14 +3,60 @@
 The purpose of this guide is to install a minimal local Armada deployment for testing and evaluation purposes.
 
 ## Pre-requisites
-- Linux OS
+
 - Git
 - Docker
 - Helm v3
 - Kind
 - Kubectl
 
-N.B.  Ensure the current user has permission to run the `docker` command without `sudo`.
+### OS specifics
+
+#### Linux
+
+Ensure the current user has permission to run the `docker` command without `sudo`.
+
+#### macOS
+
+You can install the pre-requisites with [Homebrew](https://brew.sh):
+
+```bash
+brew cask install docker
+brew install helm kind kubernetes-cli
+```
+
+Ensure at least 5GB of RAM are allocated to the Docker VM (see Preferences -> Resources -> Advanced).
+
+#### Windows
+
+You can install the pre-requisites with [Chocolatey](https://chocolatey.org):
+
+```cmd
+choco install git docker-desktop kubernetes-helm kind kubernetes-cli
+```
+
+Ensure at least 5GB of RAM are allocated to the Docker VM (see Settings -> Resources -> Advanced).
+
+In order to follow the instructions below as-is, you should also install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
```

Comment:
I wonder if this would all just work with bash on windows (it is usually installed with git).

Reply:
Good point, I'll try it out.

Reply:
Indeed, it works fine with Git Bash, I just had to adapt the downloading of `armadactl`